controllers/message.js

```diff
@@ -2,13 +2,25 @@ const messageService = require('~/services/message')
 
 const sendMessage = async (req, res) => {
   const { id: author, role: authorRole } = req.user
-  const data = req.body
+  const { id: chat } = req.params
+  const data = { ...req.body, chat }
 
   const newMessage = await messageService.sendMessage(author, authorRole, data)
 
   res.status(201).json(newMessage)
 }
 
+const getMessages = async (req, res) => {
+  const { id: userId } = req.user
+  const { id: chatId } = req.params
```

Comment:
in sendMessage we name these varaibles as author and chat, here as userId and chatId, better use the same naming anywhere

Reply:
well, in send message we need author, when in here it can be author or receiver. So i cant name it the same

Reply:
Maybe just user and chat then

Reply:
fixed